blueprint_example/dlr-bgp.yaml

```diff
@@ -2,6 +2,7 @@ tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
   - http://www.getcloudify.org/spec/cloudify/3.4/types.yaml
+#  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-nsx-plugin/master/plugin.yaml
```

Comment:
Why are we adding commented 'code' here? For the example blueprints (especially if they're going to be converted to tests), we're probably good to use the relative-path ones from the repo.

(comment applies in other files in this PR as well)